libs/utils/utils.js

```diff
@@ -656,6 +656,21 @@ export function decorateLinks(el) {
         window.adobeIMS?.signIn();
       });
     }
+    const copyEvent = '#_evt-copy';
```

Comment:
How often is this feature used? It seems like a lot of code for something that might only be used on a couple of pages... I wonder if it would make sense to have a post-LCP method that attaches event listeners / does sth special with links?

I'm not certain that's really solvable in this PR, but I see the `decorateLinks` method growing and doubt it will stop anytime soon with the special cases.

Reply:
This feature is intended for use on the Features pages for Ps, Pr, Lr, Ai, and Id specifically for MAX, with After Effects to follow post-MAX:
[https://www.adobe.com/products/photoshop/features.html](https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.adobe.com%2Fproducts%2Fphotoshop%2Ffeatures.html&data=05%7C02%7Cdmodasara%40adobe.com%7C183a6b2ec9b54f5953f008dcc5f9be13%7Cfa7b1b5a7b34438794aed2c178decee1%7C0%7C0%7C638602923624957894%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=YQ4APsbTtnC2N7fvPxg09t0%2F7DjNkpbr4ya2Pb%2FdiLI%3D&reserved=0)
[https://www.adobe.com/products/photoshop-lightroom/features.html](https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.adobe.com%2Fproducts%2Fphotoshop-lightroom%2Ffeatures.html&data=05%7C02%7Cdmodasara%40adobe.com%7C183a6b2ec9b54f5953f008dcc5f9be13%7Cfa7b1b5a7b34438794aed2c178decee1%7C0%7C0%7C638602923624974691%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=dkdJqhDrztdzYbBQyiRPljCb3Xh3xvDCkeENOF86usg%3D&reserved=0)
[https://www.adobe.com/products/illustrator/features.html](https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.adobe.com%2Fproducts%2Fillustrator%2Ffeatures.html&data=05%7C02%7Cdmodasara%40adobe.com%7C183a6b2ec9b54f5953f008dcc5f9be13%7Cfa7b1b5a7b34438794aed2c178decee1%7C0%7C0%7C638602923624988916%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=Hx0R9AAsP2hOj5SI38ZVVM7YxFqxL7MARlQZxLd9y4k%3D&reserved=0)
[https://www.adobe.com/products/premiere/features.html](https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.adobe.com%2Fproducts%2Fpremiere%2Ffeatures.html&data=05%7C02%7Cdmodasara%40adobe.com%7C183a6b2ec9b54f5953f008dcc5f9be13%7Cfa7b1b5a7b34438794aed2c178decee1%7C0%7C0%7C638602923625001187%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=6e9aS6q4MoEjMUg%2Bw34geqy3fvjf2BDnb1pjKzTC6Y0%3D&reserved=0)
[https://www.adobe.com/products/indesign/features.html](https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.adobe.com%2Fproducts%2Findesign%2Ffeatures.html&data=05%7C02%7Cdmodasara%40adobe.com%7C183a6b2ec9b54f5953f008dcc5f9be13%7Cfa7b1b5a7b34438794aed2c178decee1%7C0%7C0%7C638602923625012513%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=4XD01AttLL8KZI8hncyqWQN%2FYPpWbsAiruuAkDvEz88%3D&reserved=0)

I am not sure if we have to time to explore alternative approaches and implement them, given that this is an urgent requirement for MAX and we need to merge this PR by 9/5. However, I do agree that the growing complexity of the `decorateLinks` method is something we should address in the future.

Reply:
Yup, just thinking out loud 😁 
As I said, in the scope of this PR fixing the use-case at hand will be fine 👍